unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/net/HttpCookieTest.scala

```diff
@@ -0,0 +1,73 @@
+package org.scalanative.testsuite.javalib.net
+
+import java.net._
+
+import org.junit.Test
+import org.junit.Assert._
+
+import org.scalanative.testsuite.utils.AssertThrows.assertThrows
+
+class HttpCookieTest {
+
+  @Test
+  def basicParseTest(): Unit = {
+
+    val out = HttpCookie.parse("set-cookie2:potato=tomato")
+    assertEquals(out.size(), 1)
+    val cookie = out.get(0)
+    assertEquals("potato", cookie.getName())
+    assertEquals("tomato", cookie.getValue())
+    assertEquals(null, cookie.getComment())
```

Comment:
A comment based on my personal style. Authors have wide latitude
to follow almost any correct style.

When writing tests, I like to take a few moments to think about
how a person running the test might figure out what went wrong and
where.

When there is only one assert in a `Test`, that is not a problem.
When there are 20 or 40 or so (some of the regex code), it can
mean many hours of tedious bi-section search.    

`assertEquals` and many of its kin have a three argument form,
where the first argument is a message.  It takes a developer 
minutes of time to place a message, but it can save a debugger hours.
Usually when debugging, time is of the essence: thing are not good and are probably going to get worse.

Here one can probably figure out the failure point if one of the 
first two fails first two.  I suspect that line 20 and kin would 
cause some puzzlement.

A quick search shows `unit-tests/shared/src/test/require-scala3-jdk12/org/scalanative/testsuite/javalib/util/stream/CollectorsTestOnJDK12.scala` following that practice.

Now meaningful messages are an ideal.  In some cases where I 
have had to figure out which of 5 or more assertions (especially
the dreaded `assertTrue` & `assertFalse`, which do not give values)
are failing and have less than zero time to do it, I fall back to 
numbering the assertions "a1", "a2", etc.  This gives me a brainless
way to distinguish them. 

Yes, I hang my head that I should be taking the time to figure out a better message.  That means figuring out what all sorts of quirky code
is supposed to be doing, where that code has nothing to do with my
presenting problem.

The downside is that when I add or delete assertions, I have to do the
tedious job of renumbering, so that later maintainers can
follow the intent.   Paying for my transgressions.

Reply:
> It takes a developer minutes of time to place a message, but it can save a debugger hours.

Lovely piece of advice :) Using `CollectorsTestOnJDK12` as a template I tried to add some similar clues to the assertions, let me know if that's what you're looking for here.